#!/usr/bin/env python
PACKAGE = "dynamic_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sat_x", double_t,    0,    "Sat x",            1,      -5.5, 5.5)
# gen.add("sat_y",       int_t,    0,    "Sat y",           1,      -5, 5)
gen.add("sat_rot",       double_t,    0,    "Sat rot",          1,      -5.5, 5.5)


exit(gen.generate(PACKAGE, "dynamic_tutorials", "Tutorials"))
